grid/app/main/routes/sfl/routes.py

```diff
@@ -53,7 +53,7 @@ def worker_cycle_request():
     return Response(response_body, status=status_code, mimetype="application/json")
 
 
-@main.route("/federated/speed-test", methods=["GET", "POST"])
+@static.route("/federated/speed-test", methods=["GET", "POST"])
```

Comment:
No need for federated part on these routes either.

Reply:
Same here, If we remove the `federated` from `/static/federated/`, then we might clash with the `static` files routes.

Reply:
Then let's change the location of the static file routes! In the future, we won't have any static files because we won't have any website running on PyGrid anymore. :)